### PR TITLE
CN 4461 community screen updates

### PR DESCRIPTION
**Pull request by srkela**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/screens/Community/Community.js [hunk 1]
```diff
@@ -36,6 +38,18 @@ class Community extends Component {
     right: "profile",
   });
 
+  constructor(props) {
+    super(props);
+
+    // const { communityStats } = this.props;
+
+    this.state = {
```
Comment (fjovakaric) [inline app/components/screens/Community/Community.js]: Don't need state if pulling from constants

File: app/components/organisms/CommunityDashboard/CommunityDashboard.js [hunk 2]
```diff
@@ -107,6 +111,19 @@ class CommunityDashboard extends Component {
...
       number = 31000;
     }
+    if (
+      name === `ASSETS AS OF ${TOTAL_ASSETS_AMOUNT.DATE}` &&
+      button === "Total AUM"
+    ) {
+      explanationText = "Total Assets Under Management";
+      number = formatter.usd(TOTAL_ASSETS_AMOUNT.TOTAL_AUM, { precision: 0 });
+    }
+    if (name === "ASSETS AS OF 11 DEC 2019" && button === "Col. & Cash") {
```
Comment (fjovakaric) [inline app/components/organisms/CommunityDashboard/CommunityDashboard.js]: use `TOTAL_ASSETS_AMOUNT.DATE`

File: app/components/organisms/CommunityDashboard/CommunityDashboard.js [hunk 3]
```diff
@@ -60,10 +58,16 @@ class CommunityDashboard extends Component {
         explanation: "Total community earn in the last 12 months",
       });
     }
+    if (name === "ASSETS AS OF 11 DEC 2019") {
```
Comment (fjovakaric) [inline app/components/organisms/CommunityDashboard/CommunityDashboard.js]: Use `TOTAL_ASSETS_AMOUNT.DATE`

File: app/components/organisms/CommunityDashboard/CommunityDashboard.js (hunk 2 above)
Comment (srkela) [reply app/components/organisms/CommunityDashboard/CommunityDashboard.js]: fixed

File: app/components/screens/Community/Community.js (hunk 1 above)
Comment (srkela) [reply app/components/screens/Community/Community.js]: fixed

File: app/components/organisms/CommunityDashboard/CommunityDashboard.js (hunk 3 above)
Comment (srkela) [reply app/components/organisms/CommunityDashboard/CommunityDashboard.js]: fixed
